javascript/javascript_in_the_real_world/linting.md

```diff
@@ -52,6 +52,6 @@ Another way to address the conflict is to use `eslint-plugin-prettier`. It lets
 
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">What is linting?</a>
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">Which problems can linting prevent?</a>
-- <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=hkfBvpEfWdA">Why should you use Prettier?</a>
+- <a target="_blank" class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">What is linting?</a>
```

Comment:
Nice spot with these @Riyan911 

We've been using markdown links for any recently added knowledge links, can we do that here as well please? 
Our lesson parser will automatically add a `target="_blank"` to any markdown links.

Example:
```suggestion
- [What is linting?](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know)

Reply:
Sure 😀. I've been trying to look for projects to contribute and came across these when I was learning from The Javascript section, so I thought to create a pull request. Thanks for the early reply though.